plover/dictionary/base.py

```diff
@@ -98,11 +101,12 @@ def __init__(self, d, filename, saver):
         self.filename = filename
         self.saver = saver
         self.lock = threading.Lock()
+        self.error_msg = ''
```

Comment:
Overall, I'd prefer "failed_entries" as a name instead of "error_msg". It should be a list.

Initially:

`self.failed_entries = None`